user_data.sh

```diff
@@ -197,4 +197,26 @@ cat > ~/mycron << EOF
 0 0 * * * yum -y update --security
 EOF
 crontab ~/mycron
-rm ~/mycron
\ No newline at end of file
+rm ~/mycron
+
+%{ if onelogin_sync }
+
+cat > /usr/bin/bastion/onelogin_sync.py << 'EOF'
+${onelogin_sync_script}
+EOF
+
+cat > /usr/bin/bastion/onelogin_sync.requirements << 'EOF'
+${onelogin_sync_requirements}
+EOF
+
+chmod 755 /usr/bin/bastion/onelogin_sync.py
+yum -y install python3
+apt-get -yq install python3
+pip3 install -r /usr/bin/bastion/onelogin_sync.requirements
```

Comment:
Isn't pip a separate package (apt python3-pip)?

Reply:
Yes, and there were a bunch of other Debian/Ubuntu issues.  Fixed now!  Thanks!

Reply:
@efredrickson-patientping can we resolve this?

Reply:
Yep. And I think I need to adjust my notifications.